packages/atchops/src/uuid.c

```diff
@@ -1,11 +1,36 @@
 #include "atchops/uuid.h"
 #include <atlogger/atlogger.h>
 #include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
 #include <string.h>
-#include <uuid4/uuid4.h>
 
 #define TAG "uuid"
 
+// uuid4 was originally an external library which was depended upon as
+// an external dependency, it has been moved into this file as private
+// symbols so that it's easier to make platform specific overrides
```

Comment:
Added the code from our fork of uuid4. So that it's easier to make platform specific changes, they can now happen all in one repo.

Reply:
Can we add links like

"The original uuid4 repository by rxi: https://github.com/rxi/uuid4"
"Atsign's fork of uuid4 by rxi: https://github.com/atsign-foundation/uuid4"

Reply:
I've got the license in, but sure, I'll add the link